### PR TITLE
Added dropdown in profile section of sidebar

### LINKED ISSUE #394: Add dropdown in profile section of sidebar
@amaldinesh7 _a

### PR DESCRIPTION
**Pull request by amaldinesh7**
Fixes #394 
@edwinbbu _a Please review

### CODE REVIEW

File: lib/layouts/Sidebar.js [hunk 1]
```diff
@@ -98,17 +121,30 @@ const Sidebar = ({
...
+            )}
+            isOpen={isProfileDropdownOpen}
+            position={collapsed ? "bottom-end" : "bottom"}
+            buttonProps={{
+              icon: Down,
+            }}
+            closeOnSelect={false}
+            closeOnOutsideClick={true}
+          >
+            {profileInfo?.dropdownProps?.map((item, idx) => (
```
Comment (edwinbbu) [inline lib/layouts/Sidebar.js:142]: profileInfo condition is already checked.
```suggestion
            {profileInfo.dropdownProps?.map((item, idx) => (
```

**Comment by edwinbbu** [comment]
@amaldinesh7 _a 
1. Clicking on the avatar is not opening the dropdown.
Screenshot 2021-08-02 at 10 16 07 PM

2. There is a border color on selecting the avatar which is not required.
3. Clicking outside the dropdown should close the avatar.
Screenshot 2021-08-02 at 10 17 05 PM

File: lib/layouts/Sidebar.js (hunk 1 above)
Comment (amaldinesh7) [reply lib/layouts/Sidebar.js]: Fixed

**Comment by amaldinesh7** [comment]
I will fix the UI issues later. Merging for now @edwinbbu

**Pull request merged**